.gitignore

```diff
@@ -18,9 +18,12 @@
 *.zip
 *.tar.gz
 *.rar
+target/
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
 #intellij stuff
-*.iml
\ No newline at end of file
+*.iml
+.idea/*
```

Comment:
This seems wrong, there’s plenty of IDEA stuff we want like the JSON schema file. We DONT want the libraries.

Reply:
Ok, I'll switch it